addons/mail/static/src/discuss/partner_im_status.scss

```diff
@@ -0,0 +1,13 @@
+
+$o-mail-partner-status-icon-size: 1.2em !default;
+
+.o-mail-partner-im-status {
+    width: $o-mail-partner-status-icon-size;
+    height: $o-mail-partner-status-icon-size;
+    line-height: 1.3em;
+}
+
+.o-mail-category-im-status {
+    transform: scale(0.75);
+}
```

Comment:
This style should be in `sidebar.scss`.
This is a bit annoying to deal with proper scaling based on avatar scaling...

I have a feeling we'll need to revamp this code eventually, so that we won't have to manage scaling by hand.
But ok to keep it like this for the time being. Acceptable increase of technical debt: having the feature is more important for the moment.